src/components/text-editor/__internal__/plugins/Toolbar/buttons/list.component.tsx

```diff
@@ -66,6 +67,18 @@ const ListControls = ({ namespace }: { namespace: string }) => {
     COMMAND_PRIORITY_LOW,
   );
 
+  // Recursive function to find the closest list item ancestor
```

Comment:
comment: this isn't recursive it's using a loop, it probably could be but I think the current implementation is good but the comment could be tweaked